### PR TITLE
all: setup basic travis config

### PR DESCRIPTION
**Pull request by kostya-sh**

### CODE REVIEW

**Comment by kostya-sh** [comment]
After merging this PR please go to https://travis-ci.org, login with your github account and add google/skylark repository. The build will be triggered automatically on all subsequent commits and PRs.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sudo: false
```
Comment (kevinburke) [inline .travis.yml:1]: If we're going to add this, can you add

```
go_import_path: github.com/google/skylark
```

With that, if anyone forks + pushes from their fork to Travis CI, the tests will still work.

Comment (kostya-sh) [reply .travis.yml:1]: Done, thank you for the suggestion

**Review by jayconrod — APPROVED** [review]
LGTM.

@alandonovan, You will need to enable Travis CI by following instructions at https://docs.travis-ci.com/user/getting-started/.

**Comment by alandonovan** [comment]
Thanks!

**Pull request merged**